app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -13,6 +13,7 @@ window.GOVUK.Modules.GovukAccordion = window.GOVUKFrontend.Accordion;
     this.sectionClassExpanded = 'govuk-accordion__section--expanded'
     this.sectionHeaderClass = 'govuk-accordion__section-header'
     this.sectionInnerContent = 'govuk-accordion__section-content'
+    this.showAllControls = 'govuk-accordion__show-all'
```

Comment:
Not sure it's worth declaring this as a variable when it only gets used once and the variable name is almost as long as the thing it refers to.

Reply:
Omph - I agree in isolation, however this is more sticking to convention, I felt mixing the way we select elements in the same file was potentially more confusing. I'm tempted to leave this as is for this reason.